common/src/service/ExceptionHandlingHelper.cs

```diff
@@ -11,30 +11,29 @@ namespace Microsoft.Azure.Devices
 
     using Microsoft.Azure.Devices.Common.Exceptions;
 
-    class ExceptionHandlingHelper
+    internal class ExceptionHandlingHelper
```

Comment:
explicitly calling out as internal. Should have no impact as it was already internal.